setup.cfg

```diff
@@ -1,21 +1,11 @@
-# See the docstring in versioneer.py for instructions. Note that you must
-# re-run 'versioneer.py setup' after changing this section, and commit the
-# resulting files.
-
-[versioneer]
-VCS = git
-style = pep440
-versionfile_source = folium/_version.py
-versionfile_build = folium/_version.py
-tag_prefix = v
-parentdir_prefix =
-
 [tool:pytest]
 flake8-max-line-length = 121
 flake8-ignore =
     docs/* ALL
 select = C,E,F,W,B,B950
 ignore = E203, E501, W503
+markers =
+    web: marks tests require connection (deselect with '-m "not web"')
```

Comment:
are we using this?

Reply:
Not yet but I would like to sprinkle that on some tests that fetch data. Maybe not worth it? I'll let you decide :-)

Reply:
maybe best to take that out of this PR and add it when we start using it.